app/controllers/registrations_controller.rb

```diff
@@ -183,6 +183,7 @@ def do_import
           end
         end
         registration.save!
+        registration.add_history_entry({ event_ids: registration.event_ids }, "user", current_user.id, "CSV Import")
```

Comment:
Nitpick: Are you sure Rails polymorphic associations work with lowercase `u` in `user`?
The AR model class is called `User`, I genuinely don't know whether Rails is smart enough to "skip over" the lowercase here.

Reply:
this is not a polymorphic association at all, but just a string

Reply:
But then how can the frontend correctly display the user's name instead of just the literal string `user` plus the ID `7121`?

Reply:
Oh, that's surprising. So basically the frontend completely ignores the `actor_type` column and just assumes that whatever ID is written there is also a user ID?

Reply:
You will get all of this "for free" if you set the field to a polymorphic association in Rails :wink:

Reply:
But I agree that's out of scope for this registration